department-of-veterans-affairs/caseflow
# Updates timeliness check for edit NOD date

### Description
Resolves unexpected behavior wherein users were able to set receipt date to before decision date for issues. This new check should override exemption status too.

### Acceptance Criteria

### Testing Plan
1. Go to any AMA appeal as a user that's able to edit NOD date (judge for instance)
2. Try to set NOD date to before decision date of any issue that's attached to the appeal
3. Ensure you get a timeliness error message when you try to save the new NOD date
